### ISSUE TITLE
how can i change learning rate during training?

### ISSUE THREAD

**Issue by fffupeng** [tflearn/tflearn#911]
Almost examples for tflearn use Adam to adjust learning rate during training, but how can i change learning rate while i am using sgd? I perfer sgd more than adam.

**Comment by zuoxiang95** [comment]
You can try this to crate a sgd optimizer:
sgd = SGD(learning_rate=0.01, lr_decay=0.96, decay_step=100)
regression = regression(net, optimizer=sgd)
